modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -45,9 +45,10 @@ spec:
      outputRefs:
      - elasticsearch-secure <9>
      - default <10>
+     parse: json <11>
      labels:
-       logs: application <11>
-   - name: infrastructure-audit-logs <12>
+       logs: application <12>
```

Comment:
This example is confusing - the labels and values are user-defined, but "application" is a keyword for the CLF. It will work, but readers may be confused about what is a legal label value. Suggest something like: `myLabel: myValue` or some more realistic example that doesn't use terms that are also used by the logging API.

Reply:
Updated. Thanks.